meshtastic/node.py

```diff
@@ -123,7 +123,7 @@ def requestConfig(self, configType):
             print("Requesting current config from remote node (this can take a while).")
 
         msgIndex = configType.index
-        if configType.containing_type.full_name == "LocalConfig":
+        if configType.containing_type.full_name in ("meshtastic.LocalConfig", "LocalConfig"):
```

Comment:
This is the one actual thing that just plain didn't work. The namespacing changed the full name, so trying to fetch things in LocalConfig actually fetched from ModuleConfig.